### DISCUSSION TITLE
Setting custom defaults partially ignored/overridden in latest WebUI Forge - BUG!?

### DISCUSSION THREAD

**Discussion by GhostBranchFPV** [lllyasviel/stable-diffusion-webui-forge#1924] [General]
In earlier versions of Forge I used to go to Settings > Defaults and apply the options I usually use for image generation as my defaults, to be loaded every time I open Forge. However, ever since the update that introduced Flux support to Forge, this does not work properly anymore, at least partially:

_Some_ of the options I set, e.g. **models** used, **sampling steps**, **batch count**, the **settings for hires fix** and the **settings and state of Kohya HRFix** integrated will be correctly recalled when I relaunch Forge, but other settings like the **resolution** I set as default, **sampling method** and **scheduler type**, the **on or off state of hires fix** (always turns back off) and **CFG scale** will not be properly recalled when I relaunch Forge.

Actually, when I open Forge, I can see my correct resolution and CFG scale defaults for like half a second, then it's overridden by other defaults automatically. When I look into ui-config.json, I can see the changes I made to the defaults in there, Forge is just ignoring/resetting them for some reason.

Is this a bug or am I doing something wrong?

**Comment by abzaloff** [comment]
Update Forge and change the values for each preset in the settings
![Снимок экрана 2024-09-28 105719](https://github.com/user-attachments/assets/72085856-8c2e-4050-ab5a-dbd61896f467)

**Comment by GhostBranchFPV** [comment]
Ahh! That's new! Thank your for the heads up, I found it! :)
